docs/source/docs_tech/docs_tech_arch_models.rst

```diff
@@ -64,12 +64,12 @@ There are currently three feature types and one super type implemented in CABD.
 
 Feature types:
 
-- ``barriers`` - a super feature type that includes dams, other structures, and waterfalls.
+- ``barriers`` - a super feature type that includes dams, other structures, waterfalls, and modelled crossings.
 - ``dams`` - a feature type for features classified as a dam or other structure.
 - ``waterfalls`` - a feature type for features classified as a waterfall.
 - ``fishways`` - a feature type for features classified as a fish passage structure.
-- ``medium`` - a feature type created for testing the increase in data volume expected for stream crossing data.
-- ``big`` - a feature type created for testing the increase in data volume expected for stream crossing data.
+- ``modelled crossings`` - a feature type for features classified as a modelled crossing.
+- ``stream crossings`` - a feature type for features classified as a stream crossing.
```

Comment:
Let's revise this and remove stream_crossings. Replace with strea_crossings_ncc instead. 
In the definition for modelled crossings let's define it as modelled stream crossing (i.e., ...classified as modelled stream crossing.).